### PR TITLE
Execute `test/Geo_Tests` in enso binary launcher

### PR DESCRIPTION
**Pull request by JaroslavTulach**
### Pull Request Description

Another step in making `enso` _native executable_ execute stdlib tests.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
- [x] Unit tests have been written where possible.
- [x] Benchmark [run](https://github.com/enso-org/enso/actions/runs/11495022958) is still fine

### CODE REVIEW

File: distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Java_Problems.enso [hunk 1]
```diff
@@ -26,6 +26,7 @@ polyglot java import org.enso.table.parsing.problems.AdditionalInvalidRows
 polyglot java import org.enso.table.parsing.problems.InvalidFormat
 polyglot java import org.enso.table.parsing.problems.InvalidRow
 polyglot java import org.enso.table.problems.ProblemAggregator
+polyglot java import org.enso.table.problems.ProblemAggregator.ProblemSummary
```
Comment (JaroslavTulach) [inline distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Java_Problems.enso:29]: There is a call to `ProblemSummary` instance method in this file. Importing to make the class ready for reflection access.

File: distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso [hunk 2]
```diff
@@ -2335,7 +2335,7 @@ type Column
     to_js_object self =
         name = self.java_column.getName
         storage = self.java_column.getStorage
-        storage_proxy = Array_Proxy.new storage.size i-> storage.getItem i
+        storage_proxy = Array_Proxy.new storage.size i-> storage.getItemBoxed i
```
Comment (JaroslavTulach) [inline distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso:2338]: `getItem` isn't defined in `Storage` class, but only in individual subclasses. I don't want to (I tried) import all the individual subclasses of `Storage`, so I am changing this code to call the `Storage.getItemBoxed` method. The call is reflective so boxing (in the interpreter; before compilation) is inevitable anyway.

Benchmarks shall not be affected: check [this run](https://github.com/enso-org/enso/actions/runs/11495022958).

File: build.sbt [hunk 3]
```diff
@@ -3625,19 +3626,23 @@ lazy val `engine-runner` = project
...
               "sun.font",
               "java.awt",
               "com.sun.imageio",
-              "com.sun.jna.internal.Cleaner",
-              "com.sun.jna.Structure$FFIType",
-              "akka.http"
+              "com.sun.jna",
+              "com.microsoft",
+              "akka.http",
+              "org.enso.base",
```
Comment (JaroslavTulach) [inline build.sbt:3643]: We don't want to _preinitialize stdlib classes_ in compile time. Not all of them may need to be loaded. Too bad we don't store them in common namespace prefix like `org.enso.std` - can I repackage the libraries into such common prefix?

Comment (radeusgd) [reply build.sbt:3643]: I think it should be fine to repackage like this.

File: distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso (hunk 2 above)
Comment (radeusgd) [reply distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso:2338]: > reflective so boxing (in the interpreter; before compilation) is inevitable anyway.

In the interpreter yes, but won't it make it less likely to get nicely inlined once compiled?

(Just asking out of curiosity)

**Review by Akirathan — APPROVED** [review]

File: distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso (hunk 2 above)
Comment (JaroslavTulach) [reply distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso:2338]: [Escape analysis](https://ssw.jku.at/Teaching/PhDTheses/Stadler/Thesis_Stadler_14.pdf) effectively removes all boxing in compiled code.

Comment (JaroslavTulach) [reply distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso:2338]: Executed:
```bash
enso/tools/performance/engine-benchmarks$ ./bench_download.py -s stdlib -b wip/jtulach/GeoTestsInNativeImage develop
```
and opened `enso/tools/performance/engine-benchmarks/generated_site/stdlib-benchs.html` in browser. I **don't see any problem** that should stop us from moving forward. In the worst case we can return back to calling the unboxed variant...

**Review by radeusgd — APPROVED** [review]
Approving the changes in Enso code 👍

**Review by GregoryTravis — APPROVED** [review]

**Comment by JaroslavTulach** [comment]
Now we prepare [352 classes for reflection](https://github.com/enso-org/enso/actions/runs/11494743836/job/31992809570?pr=11394#step:7:2623) by parsing the `polyglot java import` statements (used to be 200).

**Comment by JaroslavTulach** [comment]
[Geo_Tests](https://github.com/enso-org/enso/actions/runs/11494743836/job/31992809570?pr=11394#step:7:3241) are executed successfully in the CI.

**Review by jdunkerley — APPROVED** [review]

**Review by hubertp — APPROVED** [review]

File: build.sbt (hunk 3 above)
Comment (JaroslavTulach) [reply build.sbt:3643]: OK, I'll keep in mind while working on the other libraries.

**Pull request merged**